packages/test/test-end-to-end-tests/src/test/serializeAfterFailedAttach.spec.ts

```diff
@@ -5,14 +5,9 @@
 
 import { strict as assert } from "assert";
```

Comment:
this file is easier to review by hiding whitespace, as it reuses tests with slight modifications, so they get tabbed in another level